packages/node/src/plugins/RemotePublicPathRuntimeModule.js

```diff
@@ -90,7 +96,9 @@ class AutoPublicPathRuntimeModule extends RuntimeModule {
         ? `${RuntimeGlobals.publicPath} = scriptUrl;`
         : `${RuntimeGlobals.publicPath} = scriptUrl + ${JSON.stringify(
           undoPath
-        )};`
+        )};`,
+        'var addProtocol = (url)=> url.startsWith(\'//\') ? \'https:\' + url : url;',
```

Comment:
backward compat with modernjs designs. should be able to unify and rely on auto public path in near future